src/common/cloudwatch.ts

```diff
@@ -2,6 +2,7 @@ import { ICommonProps } from './';
 
 export interface ICloudwatchProps extends ICommonProps {
   function: string;
+  startTime?: number;
 }
 
-export const expectedProps = ['region', 'function', 'pattern'];
+export const expectedProps = ['region', 'function', 'startTime', 'pattern'];
```

Comment:
If `startTime` is optional you should not add it to `expectedProps` (otherwise not passing it will throw an error).
It is not visible in the test "startTime should be defaulted when not passed in" since the `common` module is mocked thus `verifyProps` doesn't throw.

If you apply the mock as follows:
```
jest.mock('../common', () => {
  const actual = jest.requireActual('../common');
  return { ...actual, epochDateMinusHours: jest.fn() };
});
```
The test will fail